src/routes/index.jsx

```diff
@@ -187,15 +187,15 @@ export default function Home() {
           {/* Chat */}
           <ContentSection id="chat" title="Chat">
             <div class="flex flex-col items-center gap-3 w-full max-w-md">
-              <p class="text-center">Join our community chat to connect with other attendees!</p>
-              <p class="text-center">For everything NixCon-related and announcements there is a Matrix room you can join at #nixcon:matrix.org</p>
+              <p class="text-center">Join our community chat to connect with other attendees! The Matrix room is the place to go for announcements and everything NixCon-related.</p>
+              <p class="text-center">If you want to support the event as a helping hand on site, we are happy to hear from you in this Matrix room. Thanks!</p>
```

Comment:
Volunteers should join a different room instead: https://matrix.to/#/#nixcon2025-orga:lassul.us

Reply:
So we need to links/buttons:
- one to the general NixCon room (mainly for participants)
- one to the NixCon-orga room (mainly for organizers, volunteers, etc.)

Reply:
Thanks for the feedback, I misunderstood during the call. I split it off into the two links and added a margin in-between. Could you both have a second look?

![20250505-1746458816-835x498](https://github.com/user-attachments/assets/3c8741aa-e38c-4424-be9c-b6d122f7b06b)